File: dhali/scripts/split.py
#!/usr/bin/env python

# import
import pandas as pd

in_file = "master-archive-clean.csv"
out_file = "new-master-archive-clean.csv"

# Load the CSV file into a DataFrame
df = pd.read_csv(in_file, dtype=str, on_bad_lines="skip")

# write cols
for column in df.columns:
    df[column].to_csv(f"{column}.csv", index=False)

File: dhali/scripts/split.r
#!/usr/bin/env Rscript

# read csv into r
df <- read.csv("masterArchive-clean-1k.csv", header = TRUE, sep = ",")

for (col in names(df)) {
    filename <- paste(col, ".csv", sep = "")

    # write to csv
    write.csv(df[, col], file = filename, row.names = FALSE)
}
